py-polars/tests/unit/test_series.py

```diff
@@ -1126,8 +1126,9 @@ def test_is_in() -> None:
     out = s.is_in([])
     assert out.to_list() == [False]  # one element?
 
-    out = s.is_in(["x", "y", "z"])
-    assert out.to_list() == [False, False, False]
+    for x_y_z in (["x", "y", "z"], {"x", "y", "z"}):
+        out = s.is_in(x_y_z)
+        assert out.to_list() == [False, False, False]
```

Comment:
I think this check would not work reliable for sets, if one of the set values would be found as the order is not determined, unlike for dicts nowadays.